libraries/AP_SerialManager/AP_SerialManager.cpp

```diff
@@ -462,11 +462,11 @@ void AP_SerialManager::init()
                     break;
 #endif
                     
-                case SerialProtocol_EFI_MS:
-                    state[i].baud = AP_SERIALMANAGER_EFI_MS_BAUD;   // update baud param in case user looks at it
+                case SerialProtocol_EFI:
+                    state[i].baud.set_default(AP_SERIALMANAGER_EFI_BAUD);
                     state[i].uart->begin(map_baudrate(state[i].baud),
-                                         AP_SERIALMANAGER_EFI_MS_BUFSIZE_RX,
-                                         AP_SERIALMANAGER_EFI_MS_BUFSIZE_TX);
+                                         AP_SERIALMANAGER_EFI_BUFSIZE_RX,
```

Comment:
And here's an example of why I'm starting to think the serial backends should be responsible for `begin(...)`.  This might not be right for all EFIs.

Reply:
they can override if they want to